apps/ledger-live-mobile/ios/ledgerlivemobile/Info.plist

```diff
@@ -10,7 +10,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>$(APP_NAME)</string>
+	<string>LL [STAGING]</string>
```

Comment:
Is this needed?

Reply:
You're right, not sure I should commit that. Either way this is replaced when doing pods build (so in the CI too). Getting rid of that change.